js/google_ima.js

```diff
@@ -52,6 +52,7 @@ require("../html5-common/js/utils/utils.js");
       //Constants
       var DEFAULT_IMA_IFRAME_Z_INDEX = 10004;
       var DEFAULT_ADS_REQUEST_TIME_OUT = 15000;
+      var DEFAULT_LOAD_VIDEO_TIME_OUT = 15000;
```

Comment:
We're just matching an IMA SDK setting to what JW Player sets as default. This is to prevent 402 errors, which FloSports saw a lot of. Doc here:
https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdsRenderingSettings.loadVideoTimeout

Error 402 doc here:
https://support.google.com/dfp_premium/answer/4442429?hl=en